lib/currency_conversion.ex

```diff
@@ -86,14 +86,27 @@ defmodule CurrencyConversion do
 
       ### Examples
 
-          iex> CurrencyConversion.get_currencies()
+          iex> #{__MODULE__}.get_currencies()
           [:EUR, :CHF, :USD]
 
       """
       @impl unquote(__MODULE__)
       def get_currencies do
         unquote(__MODULE__).get_currencies(UpdateWorker.get_rates(@update_worker))
       end
+
+      @doc """
+      Refresh exchange rates
+
+      ### Examples
+
+      iex> #{__MODULE__}.refresh_rates()
+      :ok
+      """
+      @spec refresh_rates() :: :ok | {:error, string}
+      def refresh_rates do
+        unquote(__MODULE__).refresh_rates(UpdateWorker.refresh_rates(@update_worker))
```

Comment:
The call to `unquote(__MODULE__).refresh_rates/1` is not really needed since it does nothing with the data. You can directly return the result from `UpdateWorker.refresh_rates(@update_worker)`